src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -123,15 +126,18 @@ export function initialize({
                     )
                 )
             } else {
-                await invokeLambda({
+                stats = await invokeLambda({
                     runtime,
                     ...params,
                 })
             }
 
             return getMetricDatum({
```

Comment:
Optional: Consider avoiding redundant checks on `stats` and `stats.debug` with something like:

```
const args = {
    debug: params.isDebug,
    runtime
}
if (stats && stats.debug) {
    args.debugAttachAttempts = stats.debug.attempts
    args.debugAttachDuration = stats.debug.duration
    args.debugAttachSuccess = stats.debug.success
}
```

Reply:
✔️